nautobot/core/templates/generic/object_detail.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load perms %}
+{% load plugins %}
+{% load static %}
+{% load tz %}
+
+{% block header %}
+    <div class="row noprint">
+        {% with list_url=object|validated_viewname:"list" %}
+        <div class="col-sm-8 col-md-9">
+            <ol class="breadcrumb">
+            {% block breadcrumb %}
+                <li><a href="{% url list_url %}">
+                    {{ verbose_name_plural|bettertitle }}
+                </a></li>
+                {% block extra_breadcrumb %}{% endblock extra_breadcrumb %}
+                {% if object.name %}
+                    <li><a href="{{ object.get_absolute_url }}">{{ object.name }}</a></li>
+                {% else %}
+                    <li><a href="{{ object.get_absolute_url }}">{{ object }}</a></li>
+                {% endif %}
+            {% endblock breadcrumb %}
```

Comment:
@jedelman8 Does this adequately address #590 as far as having a standard set of breadcrumbs on object-detail pages, or do you think there's still more needed to address that as separate work?

Reply:
Huh. Yeah I think it does because now plugin authors can just inherit from `generic/object_detail.html` and either overload the entire `breadcrumbs` block (I just renamed them to be plural) or just add custom breadcrumbs in between the defaults by using `extra_breadcrumbs`.

Reply:
Per discussion on Zoom, we think this is sufficient at this time to address #590.